recipes/python/python-2.7.11/libtinfo.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/configure b/configure
+index 43bf0f3..04336da 100755
+--- a/configure
++++ b/configure
+@@ -14709,7 +14709,7 @@ LIBS_no_readline=$LIBS
+ py_cv_lib_readline=no
+ { $as_echo "$as_me:${as_lineno-$LINENO}: checking how to link readline libs" >&5
+ $as_echo_n "checking how to link readline libs... " >&6; }
+-for py_libtermcap in "" tinfo ncursesw ncurses curses termcap; do
++for py_libtermcap in "" ncursesw ncurses curses termcap; do
```

Comment:
Leftover patch from from when trying to fix packageqa. The patch is removed now...